synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -139,6 +139,7 @@ void Dock_Timetrack2::changed_canvas_view_vfunc(CanvasView::LooseHandle canvas_v
 
 		update_toolbar_action();
 		set_interp_buttons_sensitivity(current_widget_timetrack->get_num_waypoints_selected());
+		toolbar->show_all();

```

Comment:
Maybe it's better just not to hide it?
I mean line 126

Reply:
Yes sure I guess that also works. Done.